ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -335,7 +354,7 @@ - (void) validatedAcquireTokenByRefreshToken:(NSString*)refreshToken
           additionalHeaders:nil
                  completion:^(NSDictionary *response)
      {
-         ADTokenCacheStoreItem* resultItem = (cacheItem) ? cacheItem : [ADTokenCacheStoreItem new];
+         ADTokenCacheStoreItem* resultItem = (cacheItem) ? [ADTokenCacheStoreItem initWithStoreItem:cacheItem] : [ADTokenCacheStoreItem new];
```

Comment:
I'm still not sure why this line is needed. Creating a copy of the cache store item shouldn't have any difference on the logic of the code.